driver/bpf/filler_helpers.h

```diff
@@ -777,7 +777,7 @@ static __always_inline int __bpf_val_to_ring(struct filler_data *data,
 		break;
 	}
 	case PT_BYTEBUF: {
-		if (val_len) {
+		if (val!=0) {
```

Comment:
I'd rather keep both check, ie: 
`val != 0 && val_len`: i think it is the safest thing to do here!

Reply:
I completely agree with you. I also added the same check in the `kmod`. Thank you @FedeDP :1st_place_medal: